API/NoDocs/Microsoft.Extensions.Time.Testing/FakeTimeProvider.cs

```diff
@@ -0,0 +1,25 @@
+// Assembly 'Microsoft.Extensions.TimeProvider.Testing'
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+namespace Microsoft.Extensions.Time.Testing;
```

Comment:
Mismatched namespace and package name.
P: Microsoft.Extensions.TimeProvider.Testing
N: Microsoft.Extensions.Time.Testing

Alternatively, TimeProvider is in the System namespace, so this could go into System.Testing?

Reply:
https://github.com/dotnet/extensions/pull/4544